source/blog/2021-03-29-dbal-2.13.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "New Release: Doctrine DBAL 2.13 with Deprecations and Forward Compatibility"
+authorName: Benjamin Eberlei
+authorEmail: kontakt@beberlei.de
+permalink: /2021/03/29/dbal-2.13.html
+---
+
+We have released the last minor version in the 2.x family of Doctrine DBAL with
```

Comment:
Never say never. I'd rephrase it with something like "there's no plan to release minor versions in the 2.x series". I believe `2.11` was the first "last" minor and we communicated it as "last planned".